docs/general/builtins/table-functions.rst

```diff
@@ -7,8 +7,7 @@ Table functions
 ===============
 
 Table functions are :ref:`functions <gloss-function>` that produce a set of
-rows. They can either be used in place of a relation in the ``FROM`` clause, or
-within the select list of a query.
+rows. They can be used in place of a relation in the ``FROM`` clause.
```

Comment:
I think this sentence should be edited because it could be misinterpreted that below is wrong:
```
cr> select unnest([10]);
+--------------+
| unnest([10]) |
+--------------+
|           10 |
+--------------+
```
Since `unnest` is an array function, it is not a table function if used within a select list. So, `unnest([10])` as a column name should be acceptable.